src/emailLogin/getContributors.js

```diff
@@ -0,0 +1,115 @@
+const axios = require('axios');
+const fs = require('fs');
+
+const { getDb } = require('../../db/index');
+const { logError } = require('./logUtils');
+
+const { GITHUB_ACCESS_TOKEN, GITHUB_ORG_NAME: ISOMER_GITHUB_ORG_NAME } = process.env;
+
+const ISOMER_USERS = ['isomeradmin', 'rc-davis', 'lamkeewei', 'pallani', 'LoneRifle', 'prestonlimlianjie', 'alexanderleegs', 'lisatjide', 'kwajiehao', 'gweiying', 'seaerchin', 'isomer-demo', 'NatMaeTan', ' jacksonOGP', 'chienlinggg', 'kathleenkhy', 'joshuajunmingt', 'audreytcy', 'yanjunquek', 'chloe-opengovsg', 'shazlithebestie', 'lennardl', 'oliverli', 'taufiq', 'bohpeishi', 'juliuschanjq', 'vincentopengov', 'dcshzj'];
```

Comment:
[non blocking] can we ping this form isomer core instead?

Reply:
i m not here :"(

Reply:
Shucks this is a little annoying because we need the list of historical users as well - and old members are no longer in the isomer core github team. I've added you guys, but I think this might be non-blocking anyway, since the purpose is to filter out users in the agency github team (which we've mostly stopped doing for quite a while now) - the members who are more likely to be in a agency github team are the people who were here around the time of isomer v1